nv2a-trace.py

```diff
@@ -257,6 +265,18 @@ def _parse_args():
             action="store_true",
         )
 
+        parser.add_argument(
+            "--alpha-mode",
+            default="drop",
+            choices=["drop", "keep", "both"],
+            help=(
+                "Define how the alpha channel is handled in color graphical resources.\n"
+                "  drop: Discard the alpha channel\n"
+                "  keep: Save the alpha channel\n"
+                "  drop: Save two dumps, one with the alpha channel and one without\n"
```

Comment:
Ideally, one would always dump all the channels and then fix the HTML with some simple JS (or SVG) to enable alpha changes in the HTML page (so it can show RGB-only / Alpha-only / Both active / Display individual channels / ...).
In general, I'd prefer if more of the texture decoding moved into the log viewer, so the original data can be preserved (which is interesting when looking at deferred rendering in games like Shrek, or stuff like HILO formats or RGBm for HDR in games like Wrecked).